FlashSpace/Settings/WorkspacesSettingsView.swift

```diff
@@ -30,6 +31,40 @@ struct WorkspacesSettingsView: View {
                 .foregroundStyle(.secondary)
                 .font(.callout)
             }
+
+            Section(
+                header:
+                HStack {
+                    Text("Floating Apps")
+                    Spacer()
+                    Button(action: viewModel.addFloatingApp) {
+                        Image(systemName: "plus")
+                    }
+                },
+                footer:
+                Text("Floating applications remain visible across all workspaces.")
+                    .foregroundStyle(.secondary)
+                    .font(.callout)
+            ) {
+                VStack(alignment: .leading) {
+                    List(
+                        settings.floatingApps ?? [],
+                        id: \.self
+                    ) { app in
```

Comment:
You could replace it with `ForEach`, we don't need `List` here, I think.

Reply:
fixed it